src/fighter_state.rs

```diff
@@ -1218,6 +1365,9 @@ fn throwing(
                         }
                     }
                 }
+                ItemKind::Bomb { .. } => {
+                    panic!("Can't throw bomb")
+                }
```

Comment:
It might be interesting to come back in future and make it so the player can pickup and throw the bomb before it explodes.

Reply:
For sure, I thing the only problem is that the bomb stays "fusing" for I think 0.36 seconds, so not much time to grab and throw it now, but it for sure makes some fun gameplay

Reply:
yup, I was thinking we could change the duration potentially or have it extend a bit if picked up fast enough.